substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/FallbackFeature.java

```diff
@@ -167,7 +168,11 @@ public FallbackFeature() {
             addCheck(Proxy.class.getMethod("newProxyInstance", ClassLoader.class, Class[].class, InvocationHandler.class), this::collectProxyInvokes);
 
             addCheck(System.class.getMethod("loadLibrary", String.class), this::collectJNIInvokes);
-        } catch (NoSuchMethodException e) {
+
+            Class<?> generatorClass = Class.forName("sun.reflect.MethodAccessorGenerator");
```

Comment:
Needs to be adjusted for JDK 11. Locally I changed `Package_jdk_internal_reflect` to expose this method:
```
    public static String getQualifiedName() {
        if (JavaVersionUtil.JAVA_SPEC <= 8) {
            return "sun.reflect";
        } else {
            return "jdk.internal.reflect";
        }
    }
```

Reply:
Fixed as suggested